recipe/meta.yaml

```diff
@@ -1,16 +1,15 @@
-{% set name = "cdms2" %}
 {% set version = "3.1.4" %}
 
 package:
-  name: {{ name|lower }}
+  name: cdms2
   version: {{ version }}
 
 source:
   url: https://github.com/CDAT/cdms/archive/v{{ version }}.tar.gz
   sha256: 106456b6e3c0fb05f65d23c2d259a9d995d54091f75393d2c845f0462ed8dc02
 
 build:
-  number: 5
+  number: 7
```

Comment:
@ocefpaf, I think this has already been bumped, so no need to bump again.

Reply:
Yeah. Force of habit. Let's fix that.